NOTICE.md

```diff
@@ -0,0 +1,10 @@
+## Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms of the Apache License, Version 2.0 which is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+SPDX-License-Identifier: Apache-2.0
+
+## Contributors:
+*   Robert Bosch GmbH - initial API and implementation
```

Comment:
Not sure, if we need a description about what was contributed - that can be tracked via Git history.
I (personally) would think, it's enough to list all the contributors here
```suggestion
*   Robert Bosch GmbH
```